src/value.jl

```diff
@@ -94,6 +94,8 @@ function Base.show(io::IO, x::CategoricalValue)
     end
 end
 
+Base.show(io::IO, ::MIME"text/plain", x::CategoricalValue) = Base.show(io, x)
```

Comment:
This doesn't make any difference AFAICT, since `show(io::IO, ::MIME"text/plain", x::CategoricalValue)` already calls the fallback defined in Base as `Base.show(io, x)`.

Reply:
I don't think that's true, but maybe you can give a link?

Reply:
I think what you intended is this, right?
```suggestion
Base.show(io::IO, ::MIME"text/plain", x::CategoricalValue) = show(io, get(x))
```
If we go this route, it could make sense to use a more general approach via:
```suggestion
Base.show(io::IO, m::MIME, x::CategoricalValue) = show(io, m, get(x))
```